### DISCUSSION TITLE
[Greenfield API] access to payment Summary

### DISCUSSION THREAD

**Discussion by OverTheSea1** [btcpayserver/btcpayserver#3516] [Ideas / Feature Requests]
For some "login via lightning" features and many more, there is the need to get the senders adress via the Greenfield API. 
I searched the Docs, but didn't found sth. so here's the request. 
Just a simple Api Get call to get the Summary. The Summary shown in Invoices->Details->Invoice Summary is enough. 

But please add some point where you can request the sender's adress, really important
if i just didn't find the feature feel free to inform me about my mistake
<img width="1076" alt="Bildschirmfoto 2022-03-01 um 18 45 58" src="https://user-images.githubusercontent.com/72745865/156221372-15298819-fcc9-419d-be3d-23936472dc6d.png">

**Comment by bolatovumar** [comment]
@OverTheSea1 does the `/api/v1/stores/{storeId}/invoices/{invoiceId}/payment-methods` endpoint work for you? It has most of the data that's in the "Invoice Summary" section of the invoice details page.
